### ISSUE TITLE
segmentation activation

### ISSUE THREAD

**Issue by wangshihuang** [pytorch/vision#3815]
您好，请问语义分割模型的最后一层的激活函数应该如何考虑，我看到vision/references/segmentation/train.py，并没有用到激活函数softmax，为什么？

**Comment by voldemortX** [comment]
> 您好，请问语义分割模型的最后一层的激活函数应该如何考虑，我看到vision/references/segmentation/train.py，并没有用到激活函数softmax，为什么？

First, please try use English more...

Translation to English:
Hi, could I ask what is the consideration for the last activation layer in semantic segmentation models, I see `vision/references/segmentation/train.py` does not use softmax activation fucntion, why is that?

Answer:
在PyTorch中，logsoftmax是包含在[nn.CrossEntropyLoss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html?highlight=cross%20entropy#torch.nn.CrossEntropyLoss)之内的，这样求导更方便。
In PyTorch, logsoftmax is included in [nn.CrossEntropyLoss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html?highlight=cross%20entropy#torch.nn.CrossEntropyLoss) for better numerical properties.

Refer to:
https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html?highlight=cross%20entropy#torch.nn.CrossEntropyLoss
https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html#torch.nn.NLLLoss
